test/DynamoCoreTests/MessageLogTests.cs

```diff
@@ -58,6 +59,19 @@ public void TestWarningMessageLog()
             Assert.AreEqual(ProtoCore.Runtime.WarningID.Default, warningEntry.ID);
         }
 
+        [Test]
+        public void TestInfoMessageLog()
+        {
+            string openPath = Path.Combine(TestDirectory, @"core\messagelog\ScaleInfo.dyn");
+            RunModel(openPath);
+
+            ProtoCore.RuntimeCore runtimeCore = CurrentDynamoModel.EngineController.LiveRunnerRuntimeCore;
+            Assert.AreEqual(1, runtimeCore.RuntimeStatus.InfosCount);
+
+            ProtoCore.Runtime.InfoEntry infoEntry = runtimeCore.RuntimeStatus.Infos.ElementAt(0);
+            Assert.AreEqual(ProtoCore.Runtime.InfoID.Default, infoEntry.ID);
```

Comment:
Can we add a test to update the node to use a value lying with the range, re-execute the graph and verify that the info gets cleared?

Reply:
Sure, added that in one of the existing test